### ISSUE TITLE
SVG exporting

### ISSUE THREAD

**Issue by souporserious** [oscarotero/node-sketch#1]
Hello, I just came across this wonderful library and was really intrigued by it :) I was curious if there are any plans or any current way to export SVG's with this tool? I'm looking to build a sketch babel loader and was curious if I could use this to do it?

**Comment by oscarotero** [comment]
Hi. My short term plan is building a stable and solid framework to work with sketch files, providing a simple but powerful API to find, edit, move, delete or replace any element.

Once it's done, I'd like to add extra features (not sure if they should be provided as "plugins" or something similar). SVG exporting could be a really nice feature. At a first impression, seems somewhat difficult to achieve (svg exporting is a big deal), but I have not done any research about this.

**Comment by souporserious** [comment]
Awesome, thank you for the explanation! Yeah, it seems the SVG construction will be hard :/ it'd be nice if Sketch opened up an API or small lib that could take Sketch JSON and build up an SVG somehow.

**Comment by Obooman** [comment]
Hi, reminder from October, AWESOME job need maintain

**Comment by oscarotero** [comment]
Yes. This project is still alive, but I've been a bit bussy these months.
Anyway, currently I'm open to pull requests and hope to have more time in the next months.

**Comment by pcanella** [comment]
Very late here, but I've been using `node-sketch` to extract Symbols from layer upon layer looping through `sketch.pages[0].layers` and from there using `shelljs` and [sketchtool](https://developer.sketchapp.com/guides/sketchtool/) to execute exporting to SVG (which is built into `sketchtool`). Hope that helps someone :)

**Comment by antenando** [comment]
Is there any way to export without `sketchtool` ? I try to export the slices inside the sktech file but it doesn't support the export method... can't seem to make export work without sketchtool. Thanks in advance

**Comment by sudofox** [comment]
I'm trying to use this to redo an old project that had gulp-sketch in its build pipeline, making it impossible to develop on with anything except a Mac, which I can't afford. `sketchtool`, of course, is only compatible with Mac.
